src/blocks/weather/open_weather_map.rs

```diff
@@ -386,10 +388,15 @@ impl WeatherProvider for Service<'_> {
             })
         };
 
+        let sunrise = Some(DateTime::<Utc>::from_timestamp(current_data.sys.sunrise, 0).unwrap());
```

Comment:
You don't need to unwrap the option to then wrap it in a Some.

Reply:
Well, this does ensure that it is `Some`... But it would be much better to use `.error("...")?`, of course.